src/blockchain/mod.rs

```diff
@@ -23,6 +23,18 @@ pub struct ChainToReturn {
 }
 
 impl ChainToReturn {
+    /// This function takes a `Chain` object and creates a new instance of the struct by iterating over the `blocks`
```

Comment:
This doesn't need to be as detailed - it's enough to say that this creates a data object to return via the API from a given blockchain